specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,214 @@
+                  "description":"Successful completion of the asynchronous operation",
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationResult"
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               },
+               {  
+                  "name":"operationId",
+                  "in":"path",
+                  "description":"The operation ID, which can be found from the Location field in the generate recommendation response header.",
+                  "required":true,
+                  "type":"string",
+                  "format":"uuid"
+               }
+            ]
+         }
+      },
+      "/providers/Microsoft.Subscription/subscriptionOperations":{  
+         "get":{  
+            "description":"Lists all of the available pending Microsoft.Subscription API operations.",
+            "operationId":"getAllSubscriptionOperations",
+            "responses":{  
+               "200":{  
+                  "description":"OK. The request has succeeded.",
+                  "schema":{  
+                     "$ref":"#/definitions/OperationListResult"
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               }
+            ]
+         }
+      },
+      "/{scope}/providers/Microsoft.Subscription/createSubscription":{  
+         "post":{  
+            "description":"Creates an Azure subscription",
+            "operationId":"createAzureSubscription",
+            "x-ms-long-running-operation": true,
+            "responses":{  
+               "200":{  
+                  "description":"Normal response for a successful query. The response body will contain the data that matches the filters specified in the query parameters.",
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationResult"
+                  }
+               },
+               "202":{  
+                  "description":"Accepted. Subscription creation is in progress.",
+                  "headers":{  
+                     "Location":{  
+                        "description":"GET this URL to retrieve the status of the asynchronous operation.",
+                        "type":"string"
+                     },
+                     "Retry-After":{  
+                        "description":"The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                        "type":"string"
+                     }
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "name":"scope",
+                  "in":"path",
+                  "description":"Uri path corresponding to commercial subscription or legacy subscription",
```

Comment:
"The scope of the subscription creation. Valid scopes include: providers/Microsoft.Billing/billingAccounts/{accountId}"

For now, let's not mention projects yet until AIP has defined that API and created a swagger for it.